### DISCUSSION TITLE
Mousewrites figures out how to use charturner and openpose together to create a consistent posable character

### DISCUSSION THREAD

**Discussion by Njasa2k** [lllyasviel/ControlNet#66] [General]
![mrnobody](https://user-images.githubusercontent.com/99513716/219207004-08de5aed-c56a-417f-9e25-e6d160484d60.png)
![mrnobody2](https://user-images.githubusercontent.com/99513716/219207146-b2764625-179f-4d5a-b0b9-a58a4737d0f8.png)

Charturner is a textual inversion embedding created by mousewrites for the purposes of showing multiple directions of the character.

**Comment by lllyasviel** [comment]
cool. then someone train a nerf on it
